sql/fixidx/fix_field_indexes.go

```diff
@@ -12,26 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package analyzer
+package fixidx
```

Comment:
This is an odd decision

What reason can there be for giving this its own package

Reply:
No i need to move into a new package, dependency cycles everywhere.